src/pages/settings/about/session-log/session-log.ts

```diff
@@ -146,12 +149,11 @@ export class SessionLogPage {
   }
 
   private showWarningModal() {
-    const sessionLogWarningModal = this.popupProvider.createMiniModal(
-      'sensitive-info'
-    );
-    sessionLogWarningModal.present();
-    sessionLogWarningModal.onDidDismiss(response => {
-      if (response) this.isCordova ? this.sendLogs() : this.copyToClipboard();
+    let infoSheetType: InfoSheetType = 'sensitive-info';
+    const infoSheet = this.actionSheetProvider.createInfoSheet(infoSheetType);
```

Comment:
can be simplified to `this.actionSheetProvider.createInfoSheet('sensitive-info');`. Same for all other info sheets in this diff.

Reply:
you're right! thanks